dev/import_all_classes.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import argparse
+import importlib
+import pkgutil
+import sys
+import traceback
+from inspect import isclass
+from typing import List
+
+
+def import_all_classes(
+    paths: List[str],
+    prefix: str,
+    provider_ids: List[str] = None,
+    print_imports: bool = False,
+    print_skips: bool = False,
+) -> List[str]:
+    """
+    Imports all classes in providers packages. This method loads and imports
+    all the classes found in providers, so that we can find all the subclasses
+    of operators/sensors etc.
+
+    :param paths: list of paths to look the provider packages in
+    :param prefix: prefix to add
+    :param provider_ids - provider ids that should be loaded.
+    :param print_imports - if imported class should also be printed in output
+    :param print_skips - if skipped classes should also be printed in output
+    :return: list of all imported classes
+    """
+    imported_classes = []
+    tracebacks = []
+
+    def mk_prefix(provider_id):
+        return f'{prefix}{provider_id}'
+
+    provider_prefixes = [mk_prefix(provider_id) for provider_id in provider_ids] if provider_ids else [prefix]
```

Comment:
This line is 111 characters long.


```suggestion
    if provider_ids:
        provider_prefixes = [mk_prefix(provider_id) for provider_id in provider_ids]
    else:
         provider_prefixes = [prefix]
```

Reply:
sure. more straightworward.